src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -201,12 +318,54 @@ export default class MissionWindow extends Component<ThemeProps, State> {
     this.setState({ activeVehicleMapping: newActiveVehicleMapping });
   }
 
-  private updateInformation(information: MissionInformation.Information): void {
```

Comment:
why was this moved? this used to be under `updateVehicles()`

Reply:
When i was doing `npm test` it kept telling me to move them around. Not sure why

Reply:
Oh I see, that's fine then. I'll look at the code afterwards

Reply:
You can actually keep `updateInformation()` after `updateVehicles()` and it should be fine